ioctl/client.go

```diff
@@ -199,3 +203,13 @@ func (c *client) WriteConfig(cfg config.Config) error {
 	}
 	return nil
 }
+
+func (c *client) PrintError(err error) {
+	if err := output.PrintError(err); err != nil {
+		fmt.Println("Error:", err.Error())
+	}
+}
+
+func (c *client) PrintInfo(info string) {
+	output.PrintResult(info)
```

Comment:
looks like these 2 funcs are simply calling `output.Printxxx`, then do we need to add these 2 new methods into the interface?

Reply:
yes,  the aim is test the result of cmd by gomock.